gw_eccentricity/load_data.py

```diff
@@ -1093,9 +1127,9 @@ def get_modes_dict_from_sxs_catalog_old_format(**kwargs):
             time = mode_data[:, 0]
             t = np.arange(time[0], time[-1], kwargs["deltaTOverM"])
         hlm = mode_data[:, 1] + 1j * mode_data[:, 2]
-        amp_interp = interpolate(t, time, np.abs(hlm))
-        phase_interp = interpolate(t, time, -np.unwrap(np.angle(hlm)))
-        hlm_interp = amp_interp * np.exp(-1j * phase_interp)
+        real_interp = interpolate(t, time, np.real(hlm))
+        imag_interp = interpolate(t, time, np.imag(hlm))
+        hlm_interp = real_interp + 1j * imag_interp
```

Comment:
I forget, what is the motivation for this change?

Reply:
In the extreme precession case, intepolating amp and phase on smaller timestep was introducing some artefacts where as real and imaginary was not having those issues.

Reply:
moved discussion to slack

Reply:
Added comments